### ISSUE TITLE
[BUG]: Die Hard: Vendetta- Arm twitching on both renderers, texture/mipmapping issues on HW renderer

### ISSUE THREAD

**Issue by HeroponRikiBestest** [PCSX2/pcsx2#5127]
### Describe the Bug

Both renderer issues-
-Your arm geometry twitches constantly when your gun is outstretched on both renderers.
[gs_20211209062001.gs.zip](https://github.com/PCSX2/pcsx2/files/7684869/gs_20211209062001.gs.zip)

HW Renderer issues-
- The ground texture, shadows, and other textures, such as the statue texture in the opening mission, appear too sharp on the HW renderer. Setting mipmapping to Basic or Full partially fixes this, but only somewhat; the textures still remain too sharp in comparison to the SW renderer, which handles them correctly.
HW renderer-
![gs_20211209061548](https://user-images.githubusercontent.com/50224630/145393729-34f662e3-ea6f-4fd0-bd57-a03a2f31f1a9.png)
[gs_20211209061548.gs.zip](https://github.com/PCSX2/pcsx2/files/7684898/gs_20211209061548.gs.zip)
SW renderer (Correct)-
![gs_20211209061611](https://user-images.githubusercontent.com/50224630/145393806-936cc597-e9e8-4201-98e2-0b605819fb6b.png)
[gs_20211209061611.gs.zip](https://github.com/PCSX2/pcsx2/files/7684901/gs_20211209061611.gs.zip)
- Leaves on some trees flicker on HW renderer. May be the same issue as the previous one, but I'm not certain.
[gs_20211209062001.gs.zip](https://github.com/PCSX2/pcsx2/files/7684924/gs_20211209062001.gs.zip)

Additional notes:
There seem to be some artifacts visible on the HW renderer just after the load screen ends on multiplayer, possible on singleplayer as well. I am unable to provide GSdumps, as at the moment I cannot produce a single GSdump from ctrl + shift + f8, and the timing would be too precise for me to capture it with just shift + f8. If I can resolve that issue, I will update this post with the aforementioned GSdump.

These issues were observed on both the European and German versions of the game. The only difference noticed between these versions is that swapping the renderer during a load screen makes the background disappear on the German version, while swapping renderers does nothing on the European version, and the background will only disappear on a load screen if you load a save state during it. Obviously, neither of these things are issues.

To anyone testing the game in the future: yes, the gate on the first mission is supposed to look like that, even on a real PS2. It only looks correct on the xbox version, which was probably the main target for this port.

### Reproduction Steps

All issues will be normally experienced by starting a new game and playing from there. In order to see the twitching on your arm, press circle to outstretch your gun during gameplay.

### Expected Behavior

Expected behavior was laid out in the bug descriptions.

### PCSX2 Revision

PCSX2 1.7.0-0 - compiled on Dec 9 2021 (daily ppa doesnt show git number)

### Operating System

Linux (64bit) - Specify Distro Below

### If Linux - Specify Distro

Linux Mint 20.2

### CPU

AMD Ryzen 7 PRO 4750U with Radeon Graphics

### GPU

Integrated graphics, see above

**Comment by prafullpcsx2** [comment]
Can you make a gsdump using dev 1961 or earlier so we can test old gs versions and rule out that it's a regression?

**Comment by HeroponRikiBestest** [comment]
Tested arbitrarily using https://github.com/PCSX2/pcsx2/actions/runs/1396343026. All issues appear to be present.
[gs_20211212092037.gs.zip](https://github.com/PCSX2/pcsx2/files/7698902/gs_20211212092037.gs.zip)
.

**Comment by prafullpcsx2** [comment]
https://buildbot.orphis.net/pcsx2/index.php?m=dl&rev=v1.7.0-dev-1961-g4273b6ce37&platform=windows-x86

**Comment by HeroponRikiBestest** [comment]
I don't use windows. Furthermore, that PR was made a few days before dev 1961, so it's valid.

**Comment by prafullpcsx2** [comment]
It still gives savestate too old to be read message. Anyway I will check the game myself and see what can be done.

**Comment by HeroponRikiBestest** [comment]
@prafullpcsx2 I was able to get 1961, sorry about that. Here's a gsdump; all issues still seem to be visible
[gs_20211212105738.gs.zip](https://github.com/PCSX2/pcsx2/files/7699069/gs_20211212105738.gs.zip)
.

**Comment by prafullpcsx2** [comment]
@HeroponRikiBestest Thanks. Looks like twitching has always been there on pcsx2 so it's not a regression.

**Comment by HeroponRikiBestest** [comment]
Yeah; again, sorry about getting the wrong dev build, I know what to do in the future now at least.

**Comment by Dreadmoth** [comment]
Does either of the Trilinear Filtering modes improve the appearance of the textures? (Config → Graphics Settings → Hacks → Trilinear Filtering)

**Comment by HeroponRikiBestest** [comment]
I don't know if I just didn't notice it before or it was introduced recently, but now SW mode gets some red and/or blue garbage in the bottom left. It's a little hard to notice in screenshots, but its pretty noticeable in motion.
![gs_20211231030742](https://user-images.githubusercontent.com/50224630/147820289-c02150a4-d911-42d9-a7cd-28c36de690dd.png)
[gs_20211231030742.gs.zip](https://github.com/PCSX2/pcsx2/files/7796005/gs_20211231030742.gs.zip)

**Comment by lightningterror** [comment]
Mipmapping issues should be resolved now.

**Comment by Mrlinkwii** [comment]
which issues are left here ?  since the Mipmapping issues were sloved

**Comment by refractionpcsx2** [comment]
We did some testing and it seems to be affected by rounding on VU0 and FPU

**Comment by LoStraniero91** [comment]
The twitching arm is fixed with accurate **EE Mul/Div**.
